.lando/xdebug.sh

```diff
@@ -1,10 +1,29 @@
 #!/bin/bash
 
+ALLOWED_XDEBUG_MODES=(
```

Comment:
Works well with one mode at a time. Does not work with multiple modes at once (valid requirement according to https://xdebug.org/docs/all_settings#mode):

```
lando xdebug debug,trace 
'debug,profile' is invalid mode for Xdebug. Valid modes are: https://xdebug.org/docs/all_settings#mode.
```

Reply:
Thanks, will work on that.